### PR TITLE
Improvement - PEM Linux Install guide

### PR DESCRIPTION
**Pull request by nidhibhammar**
## What Changed?

Changes are done to the PEM Linux Install guide according to PEM-4381. The last 2 steps in the description are yet to implement.

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [X] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

### CODE REVIEW

File: product_docs/docs/pem/8/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/01_prerequisites_for_installing_pem_server.mdx [hunk 1]
```diff
@@ -19,46 +19,66 @@ legacyRedirectsGenerated:
 
     The `pg_hba.conf` file on the backend database can be configured to use any supported authentication methods (for example: md5, trust,...) for connections. For information about modifying the `pg_hba.conf` file, see the [PostgreSQL core documentation](https://www.postgresql.org/docs/current/static/auth-pg-hba-conf.html).
 
-3.  If you are using a PostgreSQL database, use the following command to install the `hstore contrib module`:
+3.  Ensure that the `sslutils` extension is installed.
+
+    -   On an Advanced Server backend database, the `sslutils` extension is installed by default.
+
+    -   If you are using a PostgreSQL backend database, ensure you have access to the PostgreSQL community repository, and use the command:
```
Comment (drothery-edb) [inline product_docs/docs/pem/8/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/01_prerequisites_for_installing_pem_server.mdx:26]:
```suggestion
    -   If you are using a PostgreSQL backend database, ensure you have access to the PostgreSQL community repository. Use the applicable install command for your operating system:
```

File: product_docs/docs/pem/8/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/01_prerequisites_for_installing_pem_server.mdx [hunk 2]
```diff
@@ -19,46 +19,66 @@ legacyRedirectsGenerated:
...
     Please note that Debian 10 and Ubuntu 20 has increased the requirements to accept the certificates due to security reason. If a user wants to install the PEM Agent on any of the machines, they must upgrade `ssltuils` to 1.3 where 4096 bit RSA key and sha256 signature algorithm support has added.If the user does not upgrade `sslutils` to 1.3, then PEM Agent may fail to connect to the PEM backend database server, and it might log the error `ca md too weak`.
      
 
-
-5.  Adjust your firewall restrictions.
+4.  Adjust your firewall restrictions.
 
     If you are using a firewall, you must allow access to port `8443` on the PEM backend database:
 
+    For RHEL/Rocky Linux/AlmaLinux/OL/CentOS
```
Comment (drothery-edb) [inline product_docs/docs/pem/8/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/01_prerequisites_for_installing_pem_server.mdx:67]:
```suggestion
    For RHEL/Rocky Linux/AlmaLinux/OL/CentOS,
```

File: product_docs/docs/pem/8/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/02_installing_pem_server_and_apache_web_server_preferences.mdx [hunk 3]
```diff
@@ -5,14 +5,17 @@ legacyRedirectsGenerated:
...
 ---
 
-During the PEM server installation, you can specify your hosting preferences for the Apache Web Server(PEM-HTTPD):
+During the PEM server installation, you can specify your hosting preferences for the Apache Web Server:
 
-**To install the PEM Server and Apache Web Server (PEM-HTTPD) on the same host**
+## To install the PEM Server and Apache Web Server on the same host
 
-Follow the installation steps; while running the configuration script, select the `Web Services and Database` option to install PEM Server and Apache Web Server on the same host.
+1.  Install the PEM Server, for steps see [Installing the PEM Server on Linux](03_installing_pem_server_on_linux).
```
Comment (drothery-edb) [inline product_docs/docs/pem/8/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/02_installing_pem_server_and_apache_web_server_preferences.mdx:12]: The link to Installing the PEM Server on Linux topic doesn't work in both sections

File: product_docs/docs/pem/8/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/02_installing_pem_server_and_apache_web_server_preferences.mdx [hunk 4]
```diff
@@ -5,14 +5,17 @@ legacyRedirectsGenerated:
...
+2.  Run the configuration script, select the `Web Services and Database` option to install PEM Server and Apache Web Server on the same host, for steps see [Configuring the PEM Server on Linux Platforms](05_configuring_the_pem_server_on_linux/#configuring_the_pem_server_on_linux).
 
-**To install the PEM Server and Apache Web Server (PEM-HTTPD) on separate hosts**
+##  To install the PEM Server and Apache Web Server on separate hosts
 
-Follow the installation steps on both the hosts. While running the configuration script, first configure the PEM Server host by selecting the `Database` option on first host and then configure an Apache Web Server (PEM-HTTPD) by selecting the `Web Services` option on the second host.
+1.  Install the PEM Server on both the hosts, for steps see [Installing the PEM Server on Linux](03_installing_pem_server_on_linux).
+2.  First configure the PEM Server host by selecting the `Database` option on the first host.
```
Comment (drothery-edb) [inline product_docs/docs/pem/8/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/02_installing_pem_server_and_apache_web_server_preferences.mdx:18]:
```suggestion
2.  Configure the PEM Server host by selecting the `Database` option on the first host. 
```

File: product_docs/docs/pem/8/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/05_configuring_the_pem_server_on_linux.mdx [hunk 5]
```diff
@@ -52,7 +51,7 @@ If you do not provide configuration properties on the command line, you will be
 3.  `Database` -Select this option to configure the PEM backend database for use by the PEM server. Please note that the specified database must reside on the local host.
 
 !!! Note
-    If the web server (PEM-HTTPD) and the backend database (PEM Server) reside on separate hosts, configure the database server first (option 3), and then web services (option 2). The script will exit if the backend database is not configured before web services.
+    If the Web Server and the backend database (PEM Server) reside on separate hosts, configure the database server first (option 3), and then web services (option 2). The script will exit if the backend database is not configured before web services.
```
Comment (drothery-edb) [inline product_docs/docs/pem/8/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/05_configuring_the_pem_server_on_linux.mdx:54]:
```suggestion
    If the Web Server and the backend database (PEM Server) reside on separate hosts, configure the database server first (option 3), and then web services (option 2). The script exits if the backend database is not configured before web services.
```

File: product_docs/docs/pem/8/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/02_installing_pem_server_and_apache_web_server_preferences.mdx (hunk 3 above)
Comment (nidhibhammar) [reply product_docs/docs/pem/8/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/02_installing_pem_server_and_apache_web_server_preferences.mdx]: Fixed the all the broken links and pushed the changes

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**